docroot/modules/custom/sitenow_pages/src/Form/SettingsForm.php

```diff
@@ -129,6 +130,25 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $featured_image_display_default ?: 'large',
     ];
 
+    $tag_display_type = $config->get('tag_display_type');
+
+    $form['global']['tag_display_type'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Display tags in pages'),
+      '#description' => $this->t('Set the default way to display a page\'s tags in the page itself.'),
+      '#options' => [
+        'do_not_display' => $this
+          ->t('Do not display tags'),
+        'tags' => $this
+          ->t('Display tag buttons'),
+        'related' => $this
+          ->t('Display related content'),
+        'tags_and_related' => $this
+          ->t('Display tag buttons and related content'),
+      ],
+      '#default_value' => $tag_display_type ?: 'do_not_display',
+    ];
```

Comment:
This doesn't have to happen in this PR, but I think this should be split out into two separate checkboxes. This is based on the idea that we are going to probably introduce other settings related to how each of these two separate features work and the checkboxes could serve as the basis for showing additional settings that are hidden otherwise.

Reply:
hey that was my idea from this AM's check-in. great minds...

Reply:
I have some ideas about this and I'm gonna record them here... soon..

Reply:
After thinking about this a bit more, I'm going to dedicate some time to decoupling tags and related features(currently "Tags List" and "Related" are two view modes on the same view, which may not be ideal) in addition to decoupling the settings fields (into checkboxes as suggested). 

This will allow for easier later expansion on both the tag buttons and the related content features. This could allow for separate view modes for related content (cards, lists of content, etc).

I think this is a better way to approach these two separate, yet "related" features and I want to make sure we reduce complexity in future updates (e.g., reducing the need for update hooks to rearrange settings).

Reply:
I'm still not 100% sold on checkboxes, but I do think the functionality needs to be separated to allow for more options down the line. @joewhitsitt, @pyrello, what do you think of two dropdowns that provide more options as they become available in the platform?

Mockup with a fictional-yet-possible alternate view for related content:

<img width="538" alt="Screen Shot 2022-03-04 at 1 17 38 PM" src="https://user-images.githubusercontent.com/472923/156827730-4411ae15-bcc2-4b1e-9394-23e25e862617.png">

<img width="582" alt="Screen Shot 2022-03-04 at 1 22 08 PM" src="https://user-images.githubusercontent.com/472923/156828149-110a47e7-f47d-42aa-8e65-32d8fec3c790.png">

Reply:
So, I think when it comes to "related content," yeah it makes sense to split this off. If you are saying that the headings/bulleted list (grouped by tag) option is no longer to be associated with tags and instead act as an either/or to content as cards with images, then I am fine with this change. Keep in mind that splitting off the "public display of tag information" into all of these buckets makes the future of hiding tags site-wide more complicated: https://github.com/uiowa/uiowa/issues/3420

Reply:
@joewhitsitt Thanks for bringing up the hiding tags option. I'd heard about "rabbit hole" here-and-there from dropping into meetings but didn't have a sense of what it was 'til now.

That's definitely something I'm going to keep in mind going forward. I think we can work together on making sure a future option to disable the taxonomy term views doesn't conflict with the options presented in this PR. I'm more than willing to make adjustments to the views and the logic associated with the settings form if someone has the taxonomy term view disabled.

Reply:
The reason I suggested checkboxes is because each checkbox can serve as the trigger to show more options related to each feature. If the box is checked, those other options display. If not, they don't.

I am imagining something like:
```
[ ] Display tags

[ ] Display related content
```

If the user checks the first box, it would look more like:
```
[x] Display tags

Display tags as
---
Cards with images
Headings with bulleted lists
---
(^ this is a select list)

[ ] Display related content
```